src/main/java/com/google/gwt/site/webapp/client/GWTProjectEntryPoint.java

```diff
@@ -122,68 +144,96 @@ public void f(Element e) {
   private void openMenu() {
     // close all submenus
     // todo hide first anchor with css
-    $("#submenu > nav > ul > li").hide().children("a").hide();
+    forEach("#submenu > nav > ul > li", el -> {
+      el.style.display = "none";
+      forEachChild(el, "a", child -> child.style.display = "none");
+    });
 
     String path = Window.Location.getPath();
-    GQuery selectedItem = $("#submenu a[href='" + path + "']")
-        .filter(new Predicate() {
-          @Override
-          public boolean f(Element e, int index) {
-            return !Style.Display.NONE.getCssName().equals(e.getStyle().getDisplay());
-          }
-        })
-        .eq(0);
+    HTMLElement[] container = new HTMLElement[1];
+    forEach("#submenu a[href='" + path + "']", el -> {
+      if (!el.style.display.equals("none")) {
+        container[0] = el;
```

Comment:
Technically this is changing the behavior isn't it? instead of selecting the first item that doesn't have `display:none`, this returns the last one (instead of stopping the loop early)?